error.go

```diff
@@ -2,6 +2,15 @@ package appcatalog
 
 import "github.com/giantswarm/microerror"
 
+var invalidTarballError = &microerror.Error{
+	Kind: "executionFailed",
```

Comment:
```suggestion
	Kind: "invalidTarballError",
```

Definitely. The error needs to be aligned with `fmt` but what I meant was we need an error matcher.